faiss.py

```diff
@@ -0,0 +1,39 @@
+import pickle
+import faiss
+import abc
+
+class FaissIndexer:
+  
+  @abc.abstractmethod
```

Comment:
Please create interface `Indexer` with these methods, because it will be required for `DummyIndexer`.

Reply:
fixed